locales/en-US/translation.json

```diff
@@ -309,6 +321,9 @@
 			"spam": {
 				"reason": "Spam detection",
 				"reason_mentions": "Mention spam detection"
+			},
+			"username": {
+				"reason": "Banned username detection"
```

Comment:
I think it matches better Yuuki's pattern

Reply:
What about `Flagged username detection` maybe?